website/photos/management/commands/link_album_events.py

```diff
@@ -0,0 +1,111 @@
+from django.core.management.base import BaseCommand
+
+from events.models import Event
+from photos.models import Album
+
+import csv
+
+
+def write(events, albums, file):
+    for j in range(max(len(events), len(albums))):
+        if j < len(events):
+            file.write(
+                str(events[j].pk)
+                + "-"
+                + events[j].title
+                + "-"
+                + str(events[j].start.date())
+            )
+        file.write(",")
+        if j < len(albums):
+            file.write(
+                str(albums[j].pk) + "-" + albums[j].title + "-" + str(albums[j].date)
+            )
+        file.write("\n")
+
+
+class Command(BaseCommand):
+    def handle(self, *args, **options):
+        events = Event.objects.order_by("start").all()
+        albums = Album.objects.order_by("date").all()
```

Comment:
Perhaps you should filter on albums that do not have an event linked, so you can rerun the script if required.